From f551f88da0dd638f6aef3014e6ac3b44c022b6cf Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:43 -0400
Subject: [redhat] kabi: Adding symbol blk_queue_logical_block_size (block/blk-settings.c)

Message-id: <9ec6fcf3c65cb1495cd1513fc3cb9b55b994d595.1410510773.git.poros@redhat.com>
Patchwork-id: 95899
O-Subject: [PATCH RHEL7.1 043/233] kabi: Adding symbol blk_queue_logical_block_size (block/blk-settings.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to copy the characteristics of the bottom queue to top (PowerPath) queue.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_queue_logical_block_size
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_queue_logical_block_size
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_queue_logical_block_size

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_logical_block_size  b/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_logical_block_size 
new file mode 100644
index 0000000..58d6886
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_logical_block_size  
@@ -0,0 +1,2 @@
+#1-
+0x29a84f9d blk_queue_logical_block_size vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_queue_logical_block_size  b/redhat/kabi/kabi-module/kabi_s390x/blk_queue_logical_block_size 
new file mode 100644
index 0000000..6a115d0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_queue_logical_block_size  
@@ -0,0 +1,2 @@
+#1-
+0x719c0220 blk_queue_logical_block_size vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_logical_block_size  b/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_logical_block_size 
new file mode 100644
index 0000000..b94d6d3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_logical_block_size  
@@ -0,0 +1,2 @@
+#1-
+0x177c57ca blk_queue_logical_block_size vmlinux EXPORT_SYMBOL
-- 
1.7.1